strapi/strapi
# fix(content-releases): empty Details page add a button to the CM

### What does it do?

We added a new message with a button in the body of an empty Release Details page to instruct the user to add entries from the CM in the Release

### Why is it needed?

Because otherwise when you create a new release you don't have clues how to add content to it

### How to test it?

- Go to the Releases list page
- then create a new Release
- in the Details page of the new release you can see a new message that instructs the user for the following steps and a button to be redirect to the CM page
